src/components/ProductDetail.jsx

```diff
@@ -1,31 +1,50 @@
 import React, { useState, useEffect } from 'react';
-import { useParams, useHistory } from 'react-router-dom';
-import useFetch from '../hooks/useFetch';
+import { useParams, useNavigate } from 'react-router-dom';
+import { useFetch } from '../hooks/useFetch';
 import LoadingIndicator from './LoadingIndicator';
+import '../index.css';
 
 const ProductDetail = () => {
   const { id } = useParams();
-  const history = useHistory();
-  const { isLoading, data: product, error } = useFetch(`/api/products/${id}`);
+  const navigate = useNavigate();
+  const { loading, data: product, error } = useFetch(`products/${id}`);
 
-  // Handle error
-  if (error) {
-    return <div>Error: {error.message}</div>;
-  }
+  if (loading) return <div class="loading">Loading...</div>;
+  if (error) return <div>{error.message}</div>;
 
-  // Render loading indicator if data is still loading
-  if (isLoading || !product) {
-    return <LoadingIndicator />;
-  }
+  const backtoproducts = () => {
+    navigate('/products');
+  };
 
+  
+
+  
   return (
-    <div>
-      <button onClick={() => history.goBack()}>Back</button>
-      <h2>{product.name}</h2>
-      <img src={product.image} alt={product.name} />
-      <p>{product.description}</p>
-    </div>
+
+  <div className="iteminf">
+    {product && (
```

Comment:
Please use ternary syntax here instead of `&&`. This works fine in React but it can cause problems in React Native. I'd like us to get in the habit of using the ternary instead.

```
 {product ? ( <YourJSX /> ) : null 
```